Datos guardados exitosamente en '/home/ruiz17/meteo/Data_storage/Historical_weather.xlsx' en la hoja '2024'.
{'temp (C)': -1.5, 'precipitation (mm/h)': 0, 'hum(%)': 91, 'pres(hPa)': 1039, 'wind_speed(m/s)': 0.93}
Traceback (most recent call last):
  File "/home/ruiz17/meteo/main.py", line 2, in <module>
    from Data_storage.storage_builder import *
  File "/home/ruiz17/meteo/Data_storage/storage_builder.py", line 3, in <module>
    from Data_handle.stadistics import *
ModuleNotFoundError: No module named 'Data_handle'
Datos guardados exitosamente en '/home/ruiz17/meteo/Data_storage/Historical_weather.xlsx' en la hoja '2024'.
{'temp (C)': -1.5, 'precipitation (mm/h)': 0, 'hum(%)': 91, 'pres(hPa)': 1040, 'wind_speed(m/s)': 1.79}
{'Mes': 'December', 'Dia': 16, 'Hora': '09:30:02', 'temp (C)': -1.13, 'precipitation (mm/h)': 0, 'hum(%)': 92, 'pres(hPa)': 1040, 'wind_speed(m/s)': 1.79}
Datos guardados exitosamente en '/home/ruiz17/meteo/Data_storage/Historical_weather.xlsx' en la hoja '2024'.
{'Mes': 'December', 'Dia': 16, 'Hora': '10:00:02', 'temp (C)': -0.31, 'precipitation (mm/h)': 0, 'hum(%)': 92, 'pres(hPa)': 1040, 'wind_speed(m/s)': 1.34}
Datos guardados exitosamente en '/home/ruiz17/meteo/Data_storage/Historical_weather.xlsx' en la hoja '2024'.
